.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+sudo: false
+node_js:
+    - "4.1"
+
+before_script:
+  #- npm install
+  - env
+
+script:
+  - npm run build
```

Comment:
They have a 'test matrix' where you can do parallel tests:

https://docs.travis-ci.com/user/speeding-up-the-build/